cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -167,6 +167,7 @@ def test_cluster_section_from_241_cfn(mocker, cfn_params_dict, expected_section_
                 "VolumeIOPS": "100, 100, 100, 100, 100",
                 "VolumeSize": "20, 20, 20, 20, 20",
                 "VolumeType": "gp2, gp2, gp2, gp2, gp2",
+                "Cores": "-1,-1",
```

Comment:
We should remove the "Cores" parameter from this test, it is the `test_cluster_section_from_241_cfn` because it is testing a set of input parameters coming from a cluster created with the 2.4.1 version.

Please add a new test on top for the 2.5.0 version with the new parameters (e.g. the IAMPolicies too)

Reply:
You missed it ;)